single-line-diagram-core/src/main/java/com/powsybl/sld/layout/LayoutParameters.java

```diff
@@ -78,6 +78,12 @@
 
     private double feederInfosIntraMargin = 10;
 
+    /**
+     * Can be used as horizontal shifting value for busInfo indicator.
+     * Shifting to the left with negative value and the right with positive value.
```

Comment:
Just say that it could be negative instead of that sentence (which is not exactly right), I think it's pretty clear as you called it margin how the shift should be done.